### DISCUSSION TITLE
`clear_override` is requiring `device_id` attribute

### DISCUSSION THREAD

**Discussion by ruimarinho** [firstof9/openevse#115] [Q&A]
Hi,

#107 added support for more granular control over `set_override` and `clear_override`. It looks like `clear_override` is expecting a `device_id` key (`required key not provided @ data['device_id']` when calling the service).

I'm not sure how `set_override` is working without setting an entity, but I'm not sure how to set `device_id` either (browsing through `.storage/` I can see an `entry_id` which could be used for that, but unsure).

**Comment by firstof9** [comment]
1. Go to dev tools -> Services
2. Select the `clear_override` service, select your openevse device
3. Press go to YAML mode
4. Copy your device id

**Comment by ruimarinho** [comment]
Got it. Why would `set_override` not need it though?

**Comment by firstof9** [comment]
It does
![image](https://user-images.githubusercontent.com/1105672/205744875-fcc124c0-6b8a-4e25-ad76-3b45963e9a18.png)
